config/wrapper/env.conf

```diff
@@ -140,4 +140,4 @@ packages =
 optional = html,varianter_yaml_to_mux
 
 [pip-package]
-package = configparser,enum
+package = configparser,enum,pep8,pylint,flake8,nose,nose2,epydoc,netifaces,pexpect,paramiko
```

Comment:
why both nose,nose2 ? something here needs to go to python2 branch?

Reply:
@sathnaga Those are needed for some of my nvme selftests.

Reply:
how about having these addressed through testcase, instead here, otherwise this list might keep increasing and will install packages which are not relevant for other tests or some env might not have them aswell...

I hope we can restrict this list w.r.t framework, rather testcase, what say?

Reply:
@sathnaga I am not sure of a method where we can handle it through test case. Any pointers on that please ?

Reply:
@sathnaga @narasimhan-v  handling pip package installation via test we need to come up with lib support which does pip package installation not sure that easy to accept. I feel that it should be part of wrapper if multiple Test uses the same pip packages as wrapper job to prepare the system so should be ok what you say ?